adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -526,4 +538,15 @@ private void doUploadMessage(final RoutingContext ctx, final String tenant, fina
             });
         }
     }
+
+    private static Integer getQoSLevel(final String qosValue) {
+        try {
+            if (qosValue == null)
```

Comment:
please use curly braces for all statements
again, I advise to use the code formatter rules provided in the `/eclipse` folder

Reply:
sure, i am using the code formatter since the last feedback. for some reason it feels to spot that